docs/install/initial_setup.rst

```diff
@@ -75,7 +75,7 @@ Here are definitions of concepts necessary to understand the available options i
 
 		    * **Learn-only device**
 
-		      Unlike a 'full device', on a learn-only device only the features for learners are available. Coaches and admins can sign in but will only see the *Learn* page. Learn-only devices also include automated data syncing functionality.
+		      Unlike a 'full device', a learn-only device will have available only the features for learners. Coaches and admins can sign in but will only see the *Learn* page. Learn-only devices also include automated data syncing functionality.
```

Comment:
slight phrasing suggestion:

```suggestion
		      Unlike a 'full device', a 'learn-only' device will only have features for learners available. Coaches and admins can sign in but will only see the *Learn* page. Learn-only devices also include automated data syncing functionality.
```

Reply:
I've been moving around the words in this sentence for a while, as that repetition: _...learn-*only* device will *only*..._ just screeches in my ears 😒

Reply:
haha!